Source/WebCore/platform/graphics/transforms/AffineTransform.cpp

```diff
@@ -327,6 +328,19 @@ Region AffineTransform::mapRegion(const Region& region) const
     return mappedRegion;
 }
 
+Path AffineTransform::mapPath(const Path& path) const
+{
+    if (isIdentityOrTranslation()) {
+        Path mappedPath(path);
+        mappedPath.translate(roundedIntSize(FloatSize(narrowPrecisionToFloat(m_transform[4]), narrowPrecisionToFloat(m_transform[5]))));
```

Comment:
Path::translate() takes FloatSize. Is there a need to create a FloatSize, rounded to IntSize and then implicitly cast it to FloatSize?